conf/app.routes

```diff
@@ -1,14 +1,13 @@
 # microservice specific routes
 
-->         /hmrc-frontend           hmrcfrontend.Routes
+->         /hmrc-frontend               hmrcfrontend.Routes
+GET        /assets/*file                controllers.Assets.versioned(path = "/public", file: Asset)
 
 GET        /                            controllers.EoriActionController.showPage
 POST       /                            controllers.EoriActionController.continueAction
 
-
 # Update EORI
 GET       /update/                      controllers.UpdateEoriController.showPage
 #POST      /update/                     controllers.UpdateEoriController.continueUpdateEori()
 
-GET        /hello                       controllers.HelloWorldController.helloWorld
-GET        /assets/*file                controllers.Assets.versioned(path = "/public", file: Asset)
+GET        /                            controllers.EoriActionController.showPage
```

Comment:
Route for EoriActionController.showPage is two times added.

Reply:
Good Catch